source/trendfiguur/03_inschatting.Rmd

```diff
@@ -0,0 +1,605 @@
+      (.data$waarheid == "mogelijke toename" & .data$score == "onduidelijk") |
+      (.data$waarheid == "matige toename" & .data$score == "hoger") |
+      (.data$waarheid == "stabiel" & .data$score == "vergelijkbaar")
+    )
+  ) -> ds_interpretatie
+ds_interpretatie |>
+  distinct(
+    .data$tweede_reeks, .data$index, .data$symbool
+  ) |>
+  complete(
+    .data$tweede_reeks, .data$index, .data$symbool
+  ) |>
+  filter(.data$tweede_reeks == 1 | .data$index == 1 & .data$symbool == 0) |>
+  bind_rows(ds_interpretatie) -> ds_interpretatie
+n_pred <- sum(is.na(ds_interpretatie$correct))
+```
+
+```{r verklaring-model}
+expand_grid(
+  intercept = "1", reeks = c("", "tweede_reeks"), index = c("", "index"),
+  symbool = c("", "symbool"),
+  id = "f(
+      id, model = 'iid',
+      hyper = list(theta = list(prior = 'pc.prec', param = c(0.5, 0.01)))
+    )"
+) |>
+  mutate(model_id = row_number()) |>
+  pivot_longer(-"model_id", names_to = "variable", values_to = "selected") |>
+  filter(.data$selected != "") |>
+  group_by(.data$model_id) |>
+  summarise(
+    formula = paste(.data$selected, collapse = " + ") |>
+      sprintf(fmt = "correct ~ %s") |>
+      map(as.formula)
+  ) |>
+  mutate(
+    model = map(
+      .data$formula,
+      ~inla(
+        .x, family = "binomial", data = ds_interpretatie,
+        control.compute = list(waic = TRUE), control.predictor = list(link = 1)
+      )
+    ),
+    waic = map(.data$model, "waic") |>
+      map_dbl("waic"),
+    waic = .data$waic - min(.data$waic),
+    weight = exp(-.data$waic / 2),
+    weight = .data$weight / sum(.data$weight)
+  ) -> model_fit
+```
+
+```{r verklaring-voorspelling}
+model_fit |>
+  transmute(
+    .data$weight,
+    predictions = map(.data$model, "summary.linear.predictor") |>
+      map(
+        ~head(.x, n_pred) |>
+        select("mean", "sd") |>
+        rownames_to_column()
+    )
+  ) |>
+  unnest("predictions") |>
+  mutate(
+    estimate = .data$weight * .data$mean,
+    variance = (.data$weight * .data$sd) ^ 2
+  ) |>
+  group_by(.data$rowname) |>
+  summarise(estimate = sum(.data$estimate), sd = sqrt(sum(.data$variance))) |>
+  mutate(
+    median = plogis(.data$estimate),
+    lcl = plogis(qnorm(0.025, .data$estimate, .data$sd)),
+    ucl = plogis(qnorm(0.975, .data$estimate, .data$sd))
+  ) |>
+  bind_cols(
+    ds_interpretatie |>
+      filter(is.na(.data$correct))
+  ) -> voorspelling
+```
+
+Tijdens de eerste reeks gaven de deelnemers in gemiddeld vier van de tien keren het juiste antwoord (figuur \@ref(fig:verklaring-beste)).
+Tijdens de tweede reeks steeg dit tot drie kwart van de gevallen.
+Vermoedelijk omdat het merendeel van de deelnemers tijdens de tweede reeks betrouwbaarheidsintervallen en referentielijnen verkoos.
+In mindere mate stellen we nog een effect vast van de schaal van de y-as: deelnemers die voor een procentuele wijziging kozen geven vaker een correct antwoord.
+Ook het tonen van de symbolen verhoogt de kans op een correct antwoord.
```

Comment:
Ik heb een beetje moeite met die zin aangezien deze causatie insinueert terwijl dit niet aangetoond is; kan puur correlatie zijn.

Reply:
Even advocaat van de duivel maar misschien was het beter geweest als de figuur die in reeks twee getoond werd niet afhankelijk was van de voorkeur van de participant om op die manier een deel van de correlatie er uit te halen?
Mensen die niet weten wat een confidence interval is (zo hebben we er eigenlijk te weinig in onze sample) zullen misschien kiezen om geen onzekerheid te tonen en zullen in reeks twee nog altijd niet in staat zijn om de figuur correct te interpreteren omdat de figuur niet de nodige info heeft. We weten echter niet of ze een figuur *met* aanduiding van onzekerheid wel hadden correct interpreteren. Misschien had dit hun nog meer in de war gebracht?